static/src/javascripts/projects/common/modules/identity/upsell/page/ConfirmEmailPage.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React from 'preact-compat';
+
+import { NewsLetterSignUps } from './NewsLetterSignUps';
+import { OptOuts } from './OptOuts';
+import { Header } from '../header/Header';
+import { AccountCreationBlock } from '../account-creation/AccountCreationBlock';
+
+type Props = {
+    csrfToken: string,
+    accountToken: ?string,
+    email: string,
+    hasPassword: boolean,
+    hasSocialLinks: boolean,
+    isUserLoggedIn: boolean,
+};
+
+export const ConfirmEmailPage = (props: Props) => {
+    let components = [<NewsLetterSignUps />, <OptOuts />];
```

Comment:
I wanted to have another function - `getComponents(props: Props)` - that was responsible for providing the components, however, I was getting the error:

> PropType is defined but prop is never used

🤔🤔🤔